bigframes/_config/__init__.py

```diff
@@ -61,10 +61,12 @@ def _init_bigquery_thread_local(self):
     @property
     def bigquery(self) -> bigquery_options.BigQueryOptions:
         """Options to use with the BigQuery engine."""
-        if self._local.bigquery_options is not None:
+        if (
+            bigquery_options := getattr(self._local, "bigquery_options", None)
```

Comment:
Not 100% sure how this was failing, but for some reason `self._local.bigquery_options` wasn't being initialized when I ran from a notebook in VS Code.

I think it's because the original `Options.__init__` from the import ran in a different thread from the one where the actual cell code was running.

Reply:
Fixed by https://github.com/googleapis/python-bigquery-dataframes/pull/741